runtime/test/mock-slot-composer.js

```diff
@@ -53,21 +66,26 @@ class MockSlotComposer extends SlotComposer {
     }
   }
 
-  initializeRecipe(recipe) {
-    super.initializeRecipe(recipe);
-  }
-
+  /** @method newExpectations()
+   * Creates a new group of expectations (that may occur in arbitrary order.
+   */
   newExpectations() {
     this.expectQueue.push([]);
     return this;
   }
 
+  /** @method expectContentItemsNumber(num, content)
+   * Returns true, if the number of items in content's model is equal to the given number.
+   */
   expectContentItemsNumber(num, content) {
     assert(content.model.items, `Content model doesn\'t have items (${num} expected}`);
     assert(content.model.items.length <= num, `Too many items (${content.model.items.length}), while only ${num} were expected.`);
     return content.model.items.length == num;
   }
 
+  /** @method expectContentItemsNumber(num, content)
```

Comment:
maybeRenderSlot, not expectContentItemsNumber

Reply:
fixed